src/config/Icon.tsx

```diff
@@ -61,6 +61,7 @@ import { sueBroom } from '../icons/SUE/broom';
 
 import { colors } from './colors';
 import { device } from './device';
+// import { Tabler } from './icons';
```

Comment:
why this?

Reply:
ahh, the default stays at Ionicons and if other icons are requested like tabler, this needs to be changed in this way. I think this should not be commited.

Reply:
or new idea, let's switch to tabler with 4.x anyways. and for releases that should stay at the old icons, we can keep them when merging, you know?

Reply:
I think it makes a lot of sense. Let's use tabs as standard from now on.  👍